apps/estuary/test/unit/estuary/services/event_retrieval_service_test.exs

```diff
@@ -1,28 +0,0 @@
-defmodule Estuary.Services.EventRetrievalServiceTest do
-  use ExUnit.Case
-  use Placebo
-
-  alias Estuary.Services.EventRetrievalService
-  alias Estuary.Query.Select
-
-  @tag capture_log: true
-  test "should return the events from the table" do
-    expected_events = [
-      %{
-        "author" => "Author-2020-01-21 23:29:20.171519Z",
-        "create_ts" => 1_579_649_360,
-        "data" => "Data-2020-01-21 23:29:20.171538Z",
-        "type" => "Type-2020-01-21 23:29:20.171543Z"
-      },
-      %{
-        "author" => "Author-2020-01-21 23:25:52.522084Z",
-        "create_ts" => 1_579_649_152,
-        "data" => "Data-2020-01-21 23:25:52.522107Z",
-        "type" => "Type-2020-01-21 23:25:52.522111Z"
-      }
-    ]
-
-    allow(Select.select_table(any()), return: {:ok, expected_events})
-    assert expected_events = EventRetrievalService.get_all()
-  end
-end
```

Comment:
Possible to update this test to verify the query that is being executed (at least so we know it's getting built with the right table name)?